js/utility/mDDuration.js

```diff
@@ -0,0 +1,7 @@
+export default function calculateDuration(startDate) {
+  const endDate = new Date().getTime();
+  if(startDate>endDate){
+    return -1;
+  }
+  return parseInt((endDate - startDate) / 86400000) + 1;
```

Comment:
named constant should be introduced for the number `86400000` so it's clear what it represents.

Reply:
86400000ms = 1 day, so I have created const `msPerDay = 86400000` constant variable